### ISSUE TITLE
Doesn't start on Fedora 33, ZSH, Node 14, yarn

### ISSUE THREAD

**Issue by sisou** [Jarred-Sumner/git-peek#7]
I installed this with `yarn global add @jarred/git-peek` on Fedora 33 Workstation, in ZSH, Node version `v14.15.4` and I get this when I run it, with or without a package name:
```
~: git-peek Jarred-Sumner/git-peek
/usr/bin/env: ‘node --no-warnings’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```
Same happens when I install with `npm install -g ...` instead.

Can I do anything, is this a configuration issue on my side?

**Comment by Jarred-Sumner** [comment]
Do you have node installed? If you type `which node`, what does it return?

**Comment by sisou** [comment]
Of course.
```bash
~: which node   
/usr/bin/node
```

**Comment by Jarred-Sumner** [comment]
> /usr/bin/env: use -[v]S to pass options in shebang lines

That's interesting! https://unix.stackexchange.com/questions/399690/multiple-arguments-in-shebang suggests I should add `-S` to the shebang.

I just published `1.1.21` can you run:
```bash
yarn global add @jarred/git-peek
```

**Comment by sisou** [comment]
That works, thanks!
